apiclient/types/template.go

```diff
@@ -2,12 +2,19 @@ package types
 
 type ProjectTemplate struct {
```

Comment:
so projectTemplate existed as a thing before?

Reply:
Nothing. It existed, but nothing depended on it and it wasn't exposed by any API.